src/formats/sprite.c

```diff
@@ -59,7 +59,7 @@ int sd_sprite_load(sd_reader *r, sd_sprite *sprite) {
     sprite->missing = sd_read_ubyte(r);
 
     // Copy sprite data, if there is any.
-    if(sprite->missing == 0) {
+    if(sprite->missing == 0 && sprite->len != 0) {
         sprite->data = omf_calloc(sprite->len, 1);
```

Comment:
While you're at it, could tou turn this omf_calloc around ? omf_calloc(1, sprite->len) seems more correct.